spec/models/miq_schedule_worker/runner_spec.rb

```diff
@@ -533,13 +530,6 @@
     it "#schedule_settings_for_ems_refresh (private)" do
       _ = ManageIQ::Providers::Microsoft::InfraManager # FIXME: Loader
 
-      stub_server_configuration(
-        :ems_refresh => {
-          :refresh_interval => 24.hours,
-          :scvmm            => {:refresh_interval => 15.minutes}
```

Comment:
I think this (or something like it) is still needed. The only reason this works still is that these happen to be the values that are set in [settings.yml](https://github.com/ManageIQ/manageiq/blob/master/config/settings.yml#L172-L174). I would prefer to make this test use something other than the defaults, but if the defaults in settings.yml are changed for some reason this test will fail.

Reply:
I will check the tests that were using this method and find the appropriate replacement.

Reply:
Thanks, it looks like this one at least is a case of a poorly framed test, but given that the PR is green it seems like that might be the case in quite a few places :worried:

Reply:
My plan was to find proper replacement for stub method and then "fix" the tests by using some intentionally ridiculous values in assertions.

Reply:
I went through tests again and replaced `stub_server_configuration` calls with `stub_settings` calls that should achieve the same effect. This caused three tests to start failing, two of which could be easily fixed. I marked last remaining failing test with a `skip` call and should be either changed to test the method behavior or the method should be fixed.